server/formatter.js

```diff
@@ -18,6 +19,12 @@ function normalizeHtml(html) {
   const $p = $('p')
   const isClean = $('meta[name="library-html-doc"]').attr('content') === '1'
 
+  let svgString
+  // load svg that will be used on images. takes ~0.2ms
+  if (html.indexOf('img') !== -1) {
+    svgString = fs.readFileSync('server/ssrComponents/expandIcon.svg', 'utf8')
```

Comment:
For file i/o, make sure to load in in at the top of the file so disk reads only needs to be done once on app boot! Alternatively, you could move this to be an asset that's loaded in an `img` tag on the front-end, or embed the svg directly in the html.

Reply:
good catch! missed the multiple reload. let me explore around for something that works while I make the other changes